### PR TITLE
Implement network selector connection status message bar.

### PR DESCRIPTION
**Pull request by BeanTosser**

### CODE REVIEW

**Comment by woodser** [comment]
- Seeing a flash of red when the app is first loaded which is jarring.
- The error bar continues to show after returning to a connected network type. For example, select stagenet (connected), mainnet (disconnected), then back to stagenet. The error bar continues to show until the connection manager next checks the connection, whereas it should check the connection immediately when the network type is switched.
- Spam on the console when syncing: "notificationBars: "

File: src/main/App.js [hunk 1]
```diff
@@ -159,7 +157,10 @@ class App extends React.Component {
     // The app should only keep tabs on the connection status of the connection manager for the current network
     this.connectionManagers["stagenet"].startCheckingConnection();  // automatically run checkConnection() at regular intervals
     // Check the connection NOW to make sure this.state.isConnectedToDaemon is INITIALLY true if the connection is up
-    this.initialState.isConnectedToDaemon = this.connectionManagers["stagenet"].isConnected(); 
+    let isConnected = this.connectionManagers["stagenet"].isConnected();
```
Comment (woodser) [inline src/main/App.js:156]: This will always use stagenet and not the selected network type.

Comment (BeanTosser) [reply src/main/App.js:156]: Incorrect. When a user changes the network, the app stops checking connection on the previously selected network and starts checking on the newly selected network.

https://github.com/BeanTosser/monerostresstester.com/blob/9d12858c23ee8c54a33652cee178a25dc0a46691/src/main/App.js#L791

**Review by woodser — CHANGES_REQUESTED** [review]
Seems to be quite stable. Only seeing these issues remaining:

- "return to home" button does not work after depositing funds
- sign out is not working

**Pull request merged**